pkg/virt-operator/install-strategy/create_test.go

```diff
@@ -278,6 +279,83 @@ var _ = Describe("Create", func() {
 		})
 	})
 
+	Context("Services", func() {
+
+		It("should patch if ClusterIp == \"\" during update", func() {
+
+			kv := &v1.KubeVirt{
+				ObjectMeta: metav1.ObjectMeta{
+					Namespace: "somenamespace",
+				},
+				Spec: v1.KubeVirtSpec{
+					ImageRegistry: "someregistery",
+					ImageTag:      "v1",
+				},
+			}
+
+			cachedService := &corev1.Service{}
+			cachedService.Spec.Type = corev1.ServiceTypeClusterIP
+			cachedService.Spec.ClusterIP = "10.10.10.10"
```

Comment:
do we need to add ipv6 test here ? or its not important ?

Reply:
not important. i could set this to anything that's != "" for the purposes here.